Closes #605 Update cdn-deploy-main.yml to be triggered on both main and 2.x (#640)
az-digital/arizona-bootstrap: README.md

[Explore Arizona Bootstrap Docs](https://digital.arizona.edu/arizona-bootstrap/)

## Content Delivery Network (CDN)
Arizona Digital maintains a CDN for easy inclusion of Arizona Bootstrap assets
in your project.

### How the CDN is updated

The CDN is updated automatically during the following scenarios:

- A pull request is merged into the `main` branch of this repository
- A pull request is merged into the `2.x` branch of this repository
- A new release is created on the `main` or `2.x` branch.

The URL structure for the CDN is as follows:
```
https://cdn.digital.arizona.edu/lib/arizona-bootstrap/<version bucket>/<asset type>/<filename>
```

#### The `<version bucket>` parameter can be any of the following
- `main` Deprecated
- `2.x` Development version of Arizona Bootstrap 2
- `5.x` Development version of Arizona Bootstrap 5
- A release version number, Example: `2.0.19` See [the release
page](https://github.com/az-digital/arizona-bootstrap/releases) for a full list
of possible options. Remove the `v` from the version number.

> - `latest-2x` The latest stable version of Arizona Bootstrap 2
> - `latest-5x` The latest stable version of Arizona Bootstrap 5

Author: ```suggestion
- `latest-2.x` The latest stable version of Arizona Bootstrap 2
- `latest-5.x` The latest stable version of Arizona Bootstrap 5
```

- `latest` Deprecated

**NOTE: `main` is deprecated and will no longer be updated as of `v2.0.20` - to
use the latest development version of the `2.x` branch use `2.x`**

**NOTE: `latest` is deprecated and will no longer be updated as of `v2.0.20` -

> to use the latest stable version of the `2.x` branch use `latest-2x`**

Author: ```suggestion
to use the latest stable version of the `2.x` branch use `latest-2.x`**
```

### Release is tagged on the `main` branch
All tagged releases on the `main` branch will:

> - Update `latest-5x` bucket

Author: ```suggestion
- Update `latest-5.x` bucket
```

- Create a new bucket for the version that was released. Example: `5.0.0`

### Release is tagged on the `2.x` branch
All tagged releases on the `2.x` branch will:

> - Update `latest-2x` bucket

Author: ```suggestion
- Update `latest-2.x` bucket
```

- `latest-5x` The latest stable version of Arizona Bootstrap 5
- `latest` Deprecated

**NOTE: `main` is deprecated and will no longer be updated as of `v2.0.20` - to
use the latest development version of the `2.x` branch use `2.x`**

**NOTE: `latest` is deprecated and will no longer be updated as of `v2.0.20` -
to use the latest stable version of the `2.x` branch use `latest-2x`**

### Release is tagged on the `main` branch
All tagged releases on the `main` branch will:
- Update `latest-5x` bucket
- Create a new bucket for the version that was released. Example: `5.0.0`

### Release is tagged on the `2.x` branch
All tagged releases on the `2.x` branch will:
- Update `latest-2x` bucket
- Create a new bucket for the version that was released. Example: `2.0.20`

### Pull request is merged into `main` branch
All code merged into the `main` branch will update `5.x` bucket.

### Pull request is merged into `2.x` branch
All code merged into the `2.x` branch will update the `2.x` bucket.

#### The `<asset type>` parameter can be any of the following
- `css`
- `js`

#### The `<filename>` parameter is the specific file you are attempting to use.

> 

Author: ```suggestion
```

page](https://github.com/az-digital/arizona-bootstrap/releases) for a full list
of possible options. Remove the `v` from the version number.
- `latest-2x` The latest stable version of Arizona Bootstrap 2
- `latest-5x` The latest stable version of Arizona Bootstrap 5
- `latest` Deprecated

**NOTE: `main` is deprecated and will no longer be updated as of `v2.0.20` - to
use the latest development version of the `2.x` branch use `2.x`**

**NOTE: `latest` is deprecated and will no longer be updated as of `v2.0.20` -
to use the latest stable version of the `2.x` branch use `latest-2x`**

### Release is tagged on the `main` branch
All tagged releases on the `main` branch will:
- Update `latest-5x` bucket
- Create a new bucket for the version that was released. Example: `5.0.0`

### Release is tagged on the `2.x` branch
All tagged releases on the `2.x` branch will:
- Update `latest-2x` bucket
- Create a new bucket for the version that was released. Example: `2.0.20`

### Pull request is merged into `main` branch
All code merged into the `main` branch will update `5.x` bucket.

### Pull request is merged into `2.x` branch
All code merged into the `2.x` branch will update the `2.x` bucket.

> 

Author: ```suggestion
```
